### PR TITLE
Updated Help description

### PR DESCRIPTION
**Pull request by pieterhelsen**
Updated the CLI's `help` command outputs to be less ambiguous in terms of which wallet to use. Specifically the help commands for `plotnft claim`, `plotnft join` and `plotnft leave`

Also updated `wallet.py` `help` commands for consistency in the use of 'Id' vs 'ID'.

### CODE REVIEW

**Comment by altendky** [comment]
Closing momentarily to rerun with recent CI updates.

File: chia/cmds/plotnft.py [hunk 1]
```diff
@@ -103,7 +105,15 @@ def create_cmd(
 
 @plotnft_cmd.command("join", short_help="Join a plot NFT to a Pool")
 @click.option("-y", "--yes", help="No prompts", is_flag=True)
-@click.option("-i", "--id", help="ID of the wallet to use", type=int, default=None, show_default=True, required=True)
+@click.option(
+    "-i",
+    "--id",
+    help="ID of the Pooling wallet to store pool rewards",
```
Comment (altendky) [inline chia/cmds/plotnft.py:111]: It feels to me like the primary consideration would be that the user is specifying the pool wallet associated with the plots they want farming for the pool.  The rewards going to that wallet seems more like an effect than a goal.

File: chia/cmds/plotnft.py [hunk 2]
```diff
@@ -134,7 +144,15 @@ def join_cmd(wallet_rpc_port: Optional[int], fingerprint: int, id: int, fee: int
 
 @plotnft_cmd.command("leave", short_help="Leave a pool and return to self-farming")
 @click.option("-y", "--yes", help="No prompts", is_flag=True)
-@click.option("-i", "--id", help="ID of the wallet to use", type=int, default=None, show_default=True, required=True)
+@click.option(
+    "-i",
+    "--id",
+    help="ID of the Pooling wallet currently used to store pool rewards for the plotnft",
```
Comment (altendky) [inline chia/cmds/plotnft.py:150]: As above.

**Review by altendky — CHANGES_REQUESTED** [review]
Thanks for the contribution and my apologies for the delayed review here.  I definitely appreciate the improved consistency and clarity.

**Comment by emlowe** [comment]
Closing PR

**Pull request closed without merging**